.github/dependabot.yml

```diff
@@ -1,11 +1,18 @@
-# To get started with Dependabot version updates, you'll need to specify which
-# package ecosystems to update and where the package manifests are located.
-# Please see the documentation for all configuration options:
-# https://docs.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
-
 version: 2
 updates:
-  - package-ecosystem: "gomod" # See documentation for possible values
-    directory: "/" # Location of package manifests
+  - directory: "/"
+    groups:
+      minor-patch:
+        update-types:
+          - "minor"
+          - "patch"
+    ignore:
+      versions: "<1"
```

Comment:
can you clarify again why we do this?

Reply:
This is an assumption I've come to accept - if a project is before v1, it hasn't settled on an API, so changes are more likely to cause manual labor. I'd rather not let one or two <1 projects hold up easy merging for the rest of the upgrades, so this lets us move them into their own MRs.

Reply:
You should probably add a comment for that in the dependabot config. Sounds good to me!